dashboard/src2/components/auth/LoginBox.vue

```diff
@@ -23,42 +26,51 @@
 						{{ title }}
 					</span>
 				</div>
+				<p
+					class="mb-6 break-words text-center text-base font-normal leading-[21px] text-gray-700"
+					v-if="subtitle"
+				>
+					<template
+						v-if="typeof subtitle === 'object'"
```

Comment:
weird if condition, why is this needed?

Reply:
Was trying out multiple subtitles, but It's not needed anymore. Removing it.